pxr/usd/usdSkel/testenv/testUsdSkelValidators.cpp

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+
+#include "pxr/pxr.h"
+#include "pxr/base/tf/token.h"
+#include "pxr/usd/sdr/registry.h"
+#include "pxr/usd/usd/validationRegistry.h"
+#include "pxr/usd/usd/validationError.h"
+#include "pxr/usd/usd/validator.h"
+#include <pxr/usd/usdSkel/validatorTokens.h>
+#include <pxr/usd/usdSkel/root.h>
+#include <pxr/usd/usdSkel/bindingAPI.h>
+#include <pxr/usd/usdGeom/mesh.h>
+#include <pxr/usd/usdGeom/primvarsAPI.h>
+
+#include <algorithm>
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+void
+TestUsdSkelBindingAPIChecker()
+{
+    // Verify validator exists
+    UsdValidationRegistry &registry = UsdValidationRegistry::GetInstance();
+    const UsdValidator *validator = registry.GetOrLoadValidatorByName(
+            UsdSkelValidatorNameTokens->skelBindingApiAppliedChecker);
+    TF_AXIOM(validator);
+
+    // Create Stage and mesh with a skel binding property
+    UsdStageRefPtr usdStage = UsdStage::CreateNew("/Users/andrewbeers/test.usda");
```

Comment:
Nit: I'm sure you'll remove it but good to just write it to a temp test location instead before it's merged in, and/or just keep it in memory if you don't need to write it out.